fecgov/fec-cms
# Update search.gov index for deleted/added transition pages

## Summary (required)

- [x]  Delete transition.fec.gov  pages from search.gov ,114Y drawer.
- [x] Update search.gov index for transition.fec.gov  pages listed [here ](https://docs.google.com/document/d/1vIC_QGpDDqZL7E0qI-tlRLqhHWROLoRkNGVlnr1sREc/edit?usp=sharing)
- [x] Index the above list by pushing to search.gov 

- Resolves #3279

## Impacted areas of the application

fec/search/data/transition_pages.json
(This is the updated search index for transition)

## Screenshots
Screenshot of terminal output for deleting transiion pages from search.gov i14Y drawer
![transiton_deleted_search_index](https://user-images.githubusercontent.com/5572856/83407536-f19eba00-a3de-11ea-82f6-fa1dc6b3aff7.png)

## How to test

Confirm that global FEC search does not return the deleted transition pages listed [here ](https://docs.google.com/document/d/1vIC_QGpDDqZL7E0qI-tlRLqhHWROLoRkNGVlnr1sREc/edit?usp=sharing)
